src/color/creating_reading.js

```diff
@@ -420,35 +424,36 @@ p5.prototype.hue = function(c) {
  *
  */
 
-p5.prototype.lerpColor = function(c1, c2, amt) {
+p5.prototype.lerpColor = function() {
```

Comment:
if there's just one combination of arguments, we can leave the named arguments in the signature. if there are multiple combos, then we normally leave the parens empty. or were they removed here for another reason?